packages/babel-cli/src/babel/dir.js

```diff
@@ -31,7 +36,10 @@ export default function(commander, filenames, opts) {
         opts,
       ),
       function(err, res) {
-        if (err) return callback(err);
+        if (err) {
+          result.failure += 1;
```

Comment:
For now I gather failures count, but don't use them anywhere. Happy to remove this.

Reply:
what happens when error occurs here? i suspect it crash whole process, right? in other words - can `failure` increase beyond 1 here?

Reply:
It forwards the error object and crashes [here](https://github.com/babel/babel/pull/7439/files/ed6291a6ee05d73fb4577fbe989be9e103461e1f#diff-f93fbf6dce06b5cbf6cc65f5f7dac101L128) with non-zero exit code. I don't use `failures` anywhere now (I used to show number of failed files but decided it's not necessary, as usually babel just crash the whole thing)

Reply:
we could remove counting `failure` then and convert `result` to a "flat" counter of successes, would give future readers a clear purpose for this, with unused failures it might be a little bit unclear